File: sdkwork-backend-sdk-csharp/Models/PlusGameTournamentBatchArrangeForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusGameTournamentBatchArrangeForm
    {
        public List<int>? UserIds { get; set; }
        public List<PlusGameTournamentBatchArrangePlayerForm>? ArrangePlayers { get; set; }
        public int? SeedNoStart { get; set; }
        public int? TeamNoStart { get; set; }
        public int? GroupSize { get; set; }
        public bool? OverwriteExistingSeedNo { get; set; }
        public bool? OverwriteExistingTeamNo { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/plus_game_tournament_batch_arrange_form.go
package types

// 棋牌游戏赛事参赛者批量编排表单
type PlusGameTournamentBatchArrangeForm struct {
	UserIds []int `json:"userIds"`
	ArrangePlayers []PlusGameTournamentBatchArrangePlayerForm `json:"arrangePlayers"`
	SeedNoStart int `json:"seedNoStart"`
	TeamNoStart int `json:"teamNoStart"`
	GroupSize int `json:"groupSize"`
	OverwriteExistingSeedNo bool `json:"overwriteExistingSeedNo"`
	OverwriteExistingTeamNo bool `json:"overwriteExistingTeamNo"`
}
